src/Core/Cache/Type/APCuCache.php

```diff
@@ -23,20 +23,19 @@
 
 use Friendica\Core\Cache\Enum\Duration;
 use Friendica\Core\Cache\Capability\ICanCacheInMemory;
-use Friendica\Core\Cache\Enum\Type;
 use Friendica\Core\Cache\Exception\InvalidCacheDriverException;
 
 /**
  * APCu Cache.
  */
 class APCuCache extends AbstractCache implements ICanCacheInMemory
 {
+	public static $NAME = 'apcu';
```

Comment:
If it's not a constant, please use `snakeCase`.
```suggestion
	const NAME = 'apcu';
```
```suggestion
	public static $name = 'apcu';
```